python/xgrammar/compiler.py

```diff
@@ -196,3 +214,16 @@ def compile_grammar(
     def clear_cache(self) -> None:
         """Clear all cached compiled grammars."""
         self._handle.clear_cache()
+
+    @property
+    def memory_usage_bytes(self) -> int:
```

Comment:
```suggestion
    def get_cache_size_bytes(self) -> int:
```
don't set it as a property